### ISSUE TITLE
Wrong computation of Chest accelerometers offsets

### ISSUE THREAD

**Issue by nunoguedelha** [nunoguedelha/jointOffsetCalibInertial#4]
When calibrating the accelerometers mounted on the chest (**9b7, 9b8, 9b9, 9b10**), the computed offsets magnitude is too high:
```
obj.calibrationMap.values({'chest_mtb_acc_9b10','chest_mtb_acc_9b7','chest_mtb_acc_9b8','chest_mtb_acc_9b9'})
       9B7        9B8        9B9       9B10
   -6.6668   -27.2880   -16.1951    -7.8193
   -0.5873    -3.2391     4.3077    -5.3774
   -0.9743    -5.9639    31.2647   -14.2950
```

This is causing the angle error between the measured and the predicted gravity to be very high.

**Comment by nunoguedelha** [comment]
Actually the offsets and gains are wrong. The fitted ellipsoid is not the expected one, and this wrong fitting seems to be rotating the measurements as we can see in the attached figures.
![screen shot 2017-03-14 at 16 20 24](https://cloud.githubusercontent.com/assets/6848872/23907811/820b9690-08d2-11e7-9b8e-5c13e1a05164.png)
![screen shot 2017-03-14 at 16 21 13](https://cloud.githubusercontent.com/assets/6848872/23907824/87cc2400-08d2-11e7-8ce6-998e93089447.png)
![screen shot 2017-03-14 at 16 24 29](https://cloud.githubusercontent.com/assets/6848872/23907915/c2d33534-08d2-11e7-96c4-fed35642783e.png)

**Comment by nunoguedelha** [comment]
This has been fixed by commit 2ce8cec9231fca0bc4e308e600c72dd2bb0c7663. The training sensor data acquisition was intentionally suspended during the transitions between joint configurations, but this was causing the data resampling to produce glitches and wrong acceleration values.
